src/System.Collections/ref/System.Collections.cs

```diff
@@ -5,6 +5,10 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+namespace System
+{
+    public delegate TOutput Converter<in TInput, out TOutput>(TInput input);     
```

Comment:
This is going to conflict with https://github.com/dotnet/corefx/pull/10222. We should remove this from System.Collections as it is in System.Runtime.